src/asciidoc/docs/getting-started/startGuide1.adoc

```diff
@@ -212,51 +517,149 @@ state = {
 };
 ----
 
-. In the `componentDidMount` method, add the default value (`this.instantRemixing.isRemixing`) to the `setState` function, and add a listener that monitors the template for view changes.
-+
-This listener allows you to use `this.state.isRemixing` as a source of truth for the current view of the template.
+.Vanilla JS
+[.intro]
+Not needed
+
+.Angular
+[source,JavaScript]
+----
+isRemixing = false;
+title = '';
+----
+
+.Svelte
+[source,JavaScript]
+----
+let isRemixing;
+----
+--
+
+. Add a listener that monitors the template for view changes between remixing and viewing.
 +
+[.tabs,scope="lang"]
+--
+.React
+====
+[.intro]
+In the `setState` function, set the default value for `this.instantRemixing.isRemixing`, and then add the listener to maintain it as a source of truth for the current view.
+
 [source,JavaScript]
 ----
 componentDidMount() {
-this.instantRemixing = new InstantRemixing();
+  this.instantRemixing = new InstantRemixing();
+  console.log('instantRemixing', this.instantRemixing); // Confirm w/log
 
-// Set the default value for title and isRemixing state
-this.setState({
-  isRemixing: this.instantRemixing.isRemixing,
-  title: this.instantRemixing.get(['settings', 'title'])
-});
+  // Set the default value for title and isRemixing state
+  this.setState({
+    isRemixing: this.instantRemixing.isRemixing,
+    title: this.instantRemixing.get(['settings', 'title'])
+  });
 
-  // Set up a listener to update title value
-this.instantRemixing.onValueChanged((path, newValue) => {
-  if (path[0] && path[1] && path[0] === 'settings' && path[1] === 'title') {
-    this.setState({ title: newValue });
-  }
-});
+    // Set up a listener to update title value
+  this.instantRemixing.onValueChanged((path, newValue) => {
+    if (path[0] && path[1] && path[0] === 'settings' && path[1] === 'title') {
+      this.setState({ title: newValue });
+    }
+  });
+
+  // Toggle the isRemixing state based on the listener
+  this.instantRemixing.onSetRemixing((isRemixing) => {
+    this.setState({ isRemixing });
+  });
+
+  // Alert Koji we are ready to use instantRemixing
+  this.instantRemixing.ready();
+}
+----
+====
 
-// Toggle the isRemixing state based on the listener
-this.instantRemixing.onSetRemixing((isRemixing) => {
-  this.setState({ isRemixing });
+.Vanilla JS
+[source,JavaScript]
+----
+// Add a listener to handle state changes between remixing and not
+instantRemixing.onSetRemixing((isRemixing) => {
+    if (isRemixing) {
+        startRemix();
```

Comment:
It's in a later step because I had to resequence some things to make it work for all frameworks. To make this step clearer, I'm adding a note. Let me know if that helps or if there's a better solution.